coreneuron/mechanism/nrnoc_ml.ispc

```diff
@@ -166,4 +166,7 @@ struct NrnThread {
     void* mapping;
 };
 
-extern uniform double ispc_celsius;
\ No newline at end of file
+extern uniform double ispc_celsius;
+
+extern "C"
+unmasked int at_time(NrnThread* nt, varying double te);
```

Comment:
as discussed offline, considering semantics of `unmasked`, we could have something like:
```
if (something)
   at_time(...)
```

And hence we can remove unmasked from here.